ompi/mca/coll/base/coll_base_alltoall.c

```diff
@@ -80,39 +80,39 @@ mca_coll_base_alltoall_intra_basic_inplace(const void *rbuf, int rcount,
                 /* Copy the data into the temporary buffer */
```

Comment:
@bosilca (line 77 above): no need to allocate size*2 requests, we are only posting 2 requests in each turn of the loop